### PR TITLE
Add pt-BR translation for i18n

### PR DESCRIPTION
**Pull request by franzejr**

### CODE REVIEW

**Comment by graysonwright** [comment]
Anyone from the community want to help verify? :smile:

File: administrate/config/locales/administrate.pt-BR.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# encoding: UTF-8
+---
+pt-BR:
+  administrate:
+    actions:
+      confirm: Você tem certeza?
+      destroy: Deletar
+      edit: Editar
+      show: Visualizar
```
Comment (graysonwright) [inline administrate/config/locales/administrate.pt-BR.yml]: Below you use `Mostrando`, which looks like a form of `Mostrar` - what's the difference between that and `Visualizar`?

Comment (franzejr) [inline administrate/config/locales/administrate.pt-BR.yml]: Actually, ``Visualizar`` give us a sense of "View". ``Mostrar`` give us a sense of  ``displaying``, ``exhibiting``, so it makes more sense ``Mostrar/Exibir``, I've changed mostrar to exibir. It looks better!

**Comment by jayroh** [comment]
Posso dizer que tudo é legal. :+1:

Good to go, @graysonwright 

Muito obrigado @franzejr   ;)

**Comment by graysonwright** [comment]
Cool, thanks!

@franzejr - when you have a chance, can you add a `[FEATURE]` entry to `CHANGELOG.md` and squash the commits? Let's get this merged!

**Comment by franzejr** [comment]
Sure! Will do!

**Comment by rafaelsales** [comment]
@graysonwright I'm Brazilian as well and these look good!

**Comment by graysonwright** [comment]
Cool. We're waiting on #172 before we merge in any other PRs, so I'll mark this as good to go for when we start merging again. Thanks!

**Comment by graysonwright** [comment]
Merged in as https://github.com/thoughtbot/administrate/commit/0a5e2a51b2d73daf5d8d127163023a33d159b570. Thanks!

**Comment by franzejr** [comment]
:+1:

**Pull request closed without merging**